graphrag/index/api.py

```diff
@@ -0,0 +1,163 @@
+# Copyright (c) 2024 Microsoft Corporation.
+# Licensed under the MIT License
+
+"""
+Indexing API for GraphRAG.
+
+WARNING: This API is under development and may undergo changes in future releases.
+Backwards compatibility is not guaranteed at this time.
+"""
+
+import logging
+from pathlib import Path
+
+from graphrag.config.enums import CacheType, ReportingType
+from graphrag.config.models.graph_rag_config import GraphRagConfig
+from graphrag.config.resolve_timestamp_path import resolve_timestamp_path
+
+from . import create_pipeline_config as cpc
```

Comment:
This line seems a bit odd. Would this work (and get rid of line 30)?
```suggestion
from .create_pipeline_config import create_pipeline_config
```

Reply:
I felt the create_pipeline_config should be part of the main api but I am not using it directly in the api.py file. The single line version gets removed by our formatting and ruff rules since I am not using the method in the file and that is why I had to redeclare it. I can also remove it from the file if we don't think it needs to be exported with the main API.

Reply:
Nice catch!! Indeed, line 300 seems weird. Removing it!